src/test/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializerTest.java

```diff
@@ -106,7 +111,8 @@ public void givenAUniversalCreditCaseFromSyaAndMrnIsPopulated_thenUseTheIssuingO
         "DWP PIP (1),PIP,Newcastle", "DWP PIP (2),PIP,Glasgow", "DWP PIP (3),PIP,Bellevale", "DWP PIP (4),PIP,Glasgow",
         "DWP PIP (5),PIP,Springburn", "DWP PIP (6),PIP,Blackpool", "DWP PIP (7),PIP,Blackpool", "DWP PIP (8),PIP,Blackpool",
         "DWP PIP (9),PIP,Blackpool", "Inverness DRT,ESA,Inverness DRT","DWP PIP (),PIP,null",
-        "DWP PIP (11),PIP,null", "null,UC,Universal Credit", ",UC,Universal Credit", "null,PIP,null"
+        "DWP PIP (11),PIP,null", "null,UC,Universal Credit", ",UC,Universal Credit", "null,PIP,null",
+        "null,carersAllowance,Tyneview Park DRT", "DWP PIP (5),carersAllowance,Tyneview Park DRT"
```

Comment:
Is DWP PIP(5) the correct issuing office for carers allowance?

Reply:
No it's not correct issuing office. The test is defensive coding to ensure the logic picks the correct DRT based on carersAllowance, even if some reason a issuing office has been set

Reply:
ok